src/components/journal/stamps/stamps-display.tsx

```
@@ -0,0 +1,40 @@
+    <>
+      {stamps.map((item) => (
+        <Popover key={item.id}>
+          <PopoverTrigger asChild>
+            <img
+              className="z-20 w-18 h-18 absolute rounded-full hover:scale-110 transition-all cursor-pointer"
+              style={{
+                left: `${item.coordinates.x * 100}%`,
+                top: `${item.coordinates.y * 100}%`,
+              }}
+              src={`/${item.icon || 'miku'}-stamp.png`}
+            />
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)